onadata/apps/logger/data_migration/migrate_data.py

```diff
@@ -59,18 +59,29 @@ def alter_fields(self, survey_tree):
     def add_fields(self, survey_tree, new_fields):
         for field_to_add in new_fields:
```

Comment:
- I assume new_fields is a list of new fields names (tags), so maybe rename it to new_fields_tags?
- similarly, field_to_add -> field_to_add_tag

Reply:
Also in general field_name -> field_tag